x-pack/plugins/security_solution/server/endpoint/routes/metadata/metadata.test.ts

```diff
@@ -335,7 +335,7 @@ describe('test endpoint route', () => {
       expect(result.host_status).toEqual(HostStatus.ERROR);
     });
 
-    it('should return a single endpoint with status error when status is not offline or online', async () => {
+    it('should return a single endpoint with status error when status is not offline, online or enrolling', async () => {
       const response = createSearchResponse(new EndpointDocGenerator().generateHostMetadata());
```

Comment:
I assume our data generator does not need to be adjusted to use the new status, correct? (maybe it picks it up from the `enum`?)

Reply:
I don't seem to see HostStatus usage in the generator.